public/app/features/alerting/unified/components/rules/GrafanaRules.tsx

```diff
@@ -57,7 +57,8 @@ export const GrafanaRules = ({ namespaces, expandAll }: Props) => {
   const [showExportDrawer, toggleShowExportDrawer] = useToggle(false);
   const hasGrafanaAlerts = namespaces.length > 0;
 
-  const grafanaRecordingRulesEnabled = config.featureToggles.grafanaManagedRecordingRules;
+  const grafanaRecordingRulesEnabled =
```

Comment:
nit: prefer to keep this kind of logic in `useAbilities` even if it's not related to permissions.

Perhaps something like 

```
export function useGrafanaManagedRecordingRulesSupport() {
  return config.featureToggles.grafanaManagedRecordingRules && config.unifiedAlerting.recordingRulesEnabled;
}
```

Reply:
Or in `public/app/features/alerting/unified/featureToggles.ts` or combine both

Reply:
Done, thanks both